src/tracks/ui/TrackView.cpp

```diff
@@ -153,6 +153,21 @@ void TrackView::DoSetY(int y)
    mY = y;
 }
 
+int TrackView::GetTop() const
+{
+    return mY;
+}
+
+int TrackView::GetBottom() const
+{
+    return GetTop() + GetHeight();
+}
+
+int TrackView::GetActualHeight() const
+{
+    return mHeight;
```

Comment:
The existing "GetHeight" returns either this value or the minimized height, according to whether the track is now collapsed.  And that return value the "actual" height of the track as it is viewed now.

mHeight is the height the track would have if expanded (non-minimized), but the track might not be expanded now.

Perhaps the better name for this function would be `GetExpandedHeight`.

Reply:
Well it's an old name, i just moved definition to cpp file which wasn't neccessary too, would be complicated to rename this function through all the commit history though